activiti-cloud-service-common/activiti-cloud-services-common-security/src/main/java/org/activiti/cloud/identity/UserSearchParams.java

```diff
@@ -22,6 +22,7 @@ public class UserSearchParams {
     private String search;
     private Set<String> groups;
     private Set<String> roles;
+    private Set<String> types;
```

Comment:
could u user a enum?

Reply:
Done. I also added an InvalidUserTypeException which will be convenient in indentity-adapter-service.